src/sentry/static/sentry/app/components/asyncComponent.tsx

```diff
@@ -224,7 +224,7 @@ export default class AsyncComponent<
 
   remountComponent = () => {
     if (this.shouldReload) {
-      this.setState({reloading: true}, this.fetchData);
+      this.reloadData();
```

Comment:
this.reloadData sets the state but does not call this.fetchData. Is removing the callback intended?

Reply:
The `reloadData()` method calls `this.fetchData({reloading: true});` which should set state and then fetch.

Reply:
Oh yea, my bad 😬